pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/proc_publishing-a-module-in-pantheon.adoc

```diff
@@ -0,0 +1,35 @@
+include::../attributes.adoc[]
+[id="publishing-a-module-in-pantheon_{context}"]
+= Publishing a module in {ProductShortName}
+
+Publish a module in {ProductShortName} to the Customer Portal.
```

Comment:
@adahms can you please provide an insight into this?

Reply:
Hey @stoobie @theashiot - good point.

I agree - let's use an attribute for the Customer Portal to represent the publication target. Even downstream, we might end up with several sources, so it could be a good move to future-proof the docs.

Reply:
Added an attribute called PublishingPortal

Reply:
I see this change in 0a83c34f1570f154bf638f6756c1474f2a1d1d70.